package.json

```diff
@@ -34,9 +34,9 @@
     ]
   },
   "scripts": {
-    "start": "webpack-dev-server --inline --hot --content-base dist/",
+    "start": "webpack serve --inline --hot --content-base dist/",
```

Comment:
I'd check some of these options to see if they are necessary anymore.
I'd guess that content-base isn't necessary. 

Additionally there is a devServer section in webpack.config.js that seems like it can be used now instead of passing cli options:
https://webpack.js.org/configuration/dev-server/

I'm not sure if that is preferable, but perhaps it is?

Reply:
I think the `webpack.config.js` is preferable because changes will be easier to track there than in the command line of a script command in `package.json`.